model/database/wikibase_software/software_tag_model.py

```diff
@@ -0,0 +1,24 @@
+"""Wikibase Software Version Table"""
```

Comment:
should be software tag table not version table. :)

Reply:
Updated!